.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The license seems wrong

Reply:
fixed in [34c1004](https://github.com/risingwavelabs/arrow-udf/pull/42/commits/34c1004bea9224bb21317cd2a44fbf534e8a7326)